webapp/src/domain/tests/Subscriptions.tsx

```diff
@@ -47,6 +50,7 @@ export default function Subscriptions(props: SubscriptionsProps) {
     const [watchingTeams, setWatchingTeams] = useState<ReactElement[]>([])
 
     const [reloadCounter, setReloadCounter] = useState(0)
+    const [isCheckedMissingRuns, setIsCheckedMissingRuns] = useState(props.missingruns == true)
```

Comment:
In Javscript/Typescript you should always use `===` for equality comparison. Though, here I think you don't need the property at all since you always set it to `false` - you could remove it alltogether and just use `useState(false)`.